docs/guidlines/csharp.md

```diff
@@ -62,4 +62,10 @@ The source code can be found here: [Ezy.Security on Github](https://github.com/E
 
 For encryption we will use Aes256 with salt.
 This is impelemented in our Ezy.Security nuget. (TODO: Link to nuget) 
-The source code can be found here: [Ezy.Security on Github](https://github.com/EzyWebwerkstaden/Security)
\ No newline at end of file
+The source code can be found here: [Ezy.Security on Github](https://github.com/EzyWebwerkstaden/Security)
+
+### Code review guidelines
```

Comment:
Should this not be under git and codereviews instead? we already have a topic around this (not your checklist howerver)

Reply:
Agree! Moved there: https://github.com/EzyWebwerkstaden/Docs/pull/18